README.md

```diff
@@ -279,6 +279,26 @@ I would like to thank  Google for making most of the knowledge free.
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+_NOTE: MIT License
```

Comment:
- [ OPTIONAL] I think I would have been nicer to create the license file within your repository and just add  a link from here, as the would have made your readme file cleaner and professional.